### PR TITLE
Add Crowdin Integration

### PR DESCRIPTION
**Pull request by hbenali**

### CODE REVIEW

**Comment by comradekingu** [comment]
@hbenali Happy to help if you use Weblate instead. :)

**Comment by hbenali** [comment]
> @hbenali Happy to help if you use Weblate instead. :)

I'll think about it. Thank you @comradekingu for your recommendations :)

**Review by ahamdi — DISMISSED** [review]

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"project_id" : "40585"
+"api_token" : "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJ3UUVxdmhVM3ZMT2EyWGljbVV5VCIsImp0aSI6ImQ4NzlkYTBhNmI1YTQyMDU1MzdkYjBlYWYyNjA4YjM0Mjg5OTA2ODVmNjJiNWM4MzQzOGMzZGQyODUxODJjMWYwZjNmYjM1Nzc1OWI1YmMxIiwiaWF0IjoxNjM2NzAxNzI5LCJuYmYiOjE2MzY3MDE3MjksImV4cCI6MTYzOTI5MzcyOSwic3ViIjoiMjQxOTAiLCJkb21haW4iOm51bGwsInNjb3BlcyI6WyJwcm9qZWN0Il0sImFzc29jaWF0aW9ucyI6WyIqIl0sInNlc3Npb24iOjB9.Vv9VAmUAJwEb_DuQOVA4zQ9jplD3IjEleB8_o81UIk1J5uPLUAYvNd7TKI3dmJXzVQCElObNzm9Fwy65uJNZUHA4DIcMDJyZLJygtEdTvfXkkLsWe82VHFb4_dwCbK4F30eQESLj_8b67xLeDb3qWtRmkb4zBni8xKvzFpmCFwI1soxIGh8z0NDaUdPCyDvBiWBNQ7m0DY2_XNReYOkCYHAFz_T4jHrRu1Db3GwttsMWaAd0tGkElAqJWMz1QntmC34teIM2dvS5MPcNXQ6MkRo764XGDN5yrjNu03jFGiW6WddYm7w29cvpBfzlrNkpkhNsEQypxJDOThjJQ6VkSmNWUFDGcFOfJbTiki5EHjO7A0tG60ltrxhTRg1YOQCCUNyCeT5I7YYyOsZ_7R61Ww5caEFpA9AxqpWIT8wwWP1NkrwHK18Xm63KCTTqifcD8FMDh88Ls83Fiji8C35k8YLCFmFRG-mtVguUBy1rKEl09dPCD8cEUC-_RMNUaGIQklBHVzXi-Sbr-RV5O2wYXse3NQBLALUD66zL_bhFffSw5Y9KNwXj_Xb0IehyDLSgjQiQSv0OR7YjHHsTF7mTUlc37G5J2LY-zLeZ-Mg_fFjC_Jyqo2jhcve8DX8G1ZvT_DLE8haH0OQG6KWrtKtxIvpSaJGkpI0ZGuJYQbir_es"
```
Comment (ahamdi) [inline crowdin.yml:2]: @hbenali shouldn't we hide those params ?

**Comment by comradekingu** [comment]
@hbenali Happy to help set up shop too.
You can click through https://hosted.weblate.org/hosting/ and add "kingu", but the setup is very easy.
I think it is possible to use https://github.com/apps/weblate too, but never tried that approach.

**Review by ahamdi — APPROVED** [review]

**Pull request merged**